### ISSUE TITLE
Server returns http 500

### ISSUE THREAD

**Issue by hynek-urban** [windingtree/wt-read-api#191]
Example:

```
curl https://playground-api.windingtree.com/hotels/meta

{"status":500,"code":"#genericError","short":"Something went wrong.","long":"Something went wrong. Please contact the administrator."}
```

**Expected behavior**
The server should return either http 404 or another meaningful response.

**Comment by JirkaChadima** [comment]
That's probably a bad order of error handling middlewares. It's strange that it wasn't caught by tests. Thanks for reporting this.

**Comment by JirkaChadima** [comment]
Closed by b51b5ee
